android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -133,21 +133,21 @@ public void setPosition(LngLat position) {
     }
 
     /**
-     * Set the geographic position of the center of the map view
+     * Set the geographic position of the center of the map view with easing
      * @param position LngLat of the position to set
      * @param duration Time in milliseconds to ease to the given position
      */
-    public void setPosition(LngLat position, int duration) {
-        setPosition(position, duration, DEFAULT_EASE_TYPE);
+    public void setPositionEased(LngLat position, int duration) {
+        setPositionEased(position, duration, DEFAULT_EASE_TYPE);
     }
 
     /**
-     * Set the geographic position of the center of the map view
+     * Set the geographic position of the center of the map view with easing
```

Comment:
You mean the description should say "cubic easing"? This method doesn't care what easing type is used, if someone does care then they should use the other method. Plus now the docs will remain correct if we change the default easing.